test/integration/features/variables.feature

```diff
@@ -0,0 +1,60 @@
+@variables
+Feature: Variables
+  Simple scenario to test the usage of variables
+
+  Scenario: Starting Minishift
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift start" succeeds
+     Then Minishift should have state "Running"
+
+  Scenario:
+   Given Minishift has state "Running"
+    When executing "oc new-app centos/ruby-22-centos7~https://github.com/openshift/ruby-ex.git -l name=ruby-ex --name ruby-ex" succeeds
+    Then stdout should contain
+     """
+     Success
+     """
+
+  Scenario: Successfully rolled out
+   Given Minishift has state "Running" 
+    When executing "oc rollout status deploymentconfig ruby-ex --watch" succeeds
+    Then stdout should contain
+     """
+     successfully rolled out
+     """
+
+  Scenario: Successfully deployed all services
+   Given Minishift has state "Running" 
+    When executing "oc status -v" succeeds
+    Then stdout should not contain
+     """
+     has failed
+     """
+
+  Scenario: Should be able to get the pod identifier for the API gateway 
+   Given Minishift has state "Running" 
+    When set variable "gw" executing "oc get pods -o name -l name=ruby-ex"
+    Then stdout should not be empty
+     And exitcode should equal "0"
+
+  Scenario: Able to interact with the catalog from the API gateway 
+   Given Minishift has state "Running" 
+    When executing "oc rsh $(gw) cat /etc/os-release" succeeds
+    Then stdout should contain
+     """
+     CentOS Linux
+     """
+
+  Scenario: Stopping Minishift
+    Given Minishift has state "Running"
+     When executing "minishift stop" succeeds
+     Then Minishift should have state "Stopped"
+
+  Scenario: Deleting Minishift
+    Given Minishift has state "Stopped"
+     When executing "minishift delete" succeeds
+     Then Minishift should have state "Does Not Exist"
+
+  # Scenario: Gathering all information
```

Comment:
i did not intent to keep the variables.feature, but if it is useful i can remove these comments. wdyt?